beets/util/__init__.py

```diff
@@ -838,7 +854,7 @@ def plurality(objs: Sequence[T]) -> T:
     return c.most_common(1)[0]
 
 
-def convert_command_args(args: List[bytes]) -> List[str]:
+def convert_command_args(args: List[AnyStr]) -> List[str]:
```

Comment:
```suggestion
def convert_command_args(args: List[bytes | str]) -> List[str]:
```
Not entirely sure, but I'd guess that we actually use this with lists containing both `bytes` and `str`. I think `AnyStr` forces this to be a list with a fixed item type.

Reply:
To be honest `AnyStr` is only useful when it's used as one of the input parameter types and the output type - it communicates that the type stays the same. For example, `os.path.normpath`:

```python
def normpath(path: _PathLike[AnyStr]) -> AnyStr
def normpath(path: AnyStr) -> AnyStr
```

In this case there's no need for it, so I'm making the adjustment you suggested!